### ISSUE TITLE
Multiple redundant copies of the image are inserted in the resulting docx file

### ISSUE THREAD

**Issue by DidiF45** [guigrpa/docx-templates#218]
First of all thank you so much for this library. It is so powerful !!!
I encountered a problem after upgrading the version of docx-template.
I was using the version 2.7.2 with images in the docx template and the generated docx file has a correct size (6137 Ko).
After upgrading to the lastest version 4.6.4, for the same docx template I get a docx file of 84 109 Ko (13 times bigger !)
So I had a look at the xml content of the file and saw that in the new version, the file contains for example 14 times the same image (see capture1.png)
In version 2.7.2 the same image appears only once (see capture2.png).
Why are the images repeated several times in the xml content ?
![Capture1](https://user-images.githubusercontent.com/20182873/124506129-4aebd080-ddcb-11eb-903d-70a1625ebccb.PNG)
![Capture2](https://user-images.githubusercontent.com/20182873/124506141-50e1b180-ddcb-11eb-8f4d-3323b04f114d.PNG)

Thank you for your help !

**Comment by jjhbw** [comment]
Thanks for reporting. That sure is strange. A few checks just to be sure: you are using exactly the same template and `createReport` invocation? Did you use the `IMAGE` command for both templates, or did you use the older deprecated `replaceImages` API?

**Comment by DidiF45** [comment]
> Thanks for reporting. That sure is strange. A few checks just to be sure: you are using exactly the same template and `createReport` invocation? Did you use the `IMAGE` command for both templates, or did you use the older deprecated `replaceImages` API?

Thank you for your reply.
Yes I'm using the same template and the createReport invocations are as follows :
In 2.7.2 
```
var createReport = require('docx-templates');
createReport({
                template: filepath,
                data: body,
                noSandbox: true,
                output: filepath,
                literalXmlDelimiter: '||',
                processLineBreaks: true
}).then(function(report) { ... });
```

In 4.6.4
```
var createReport = require('docx-templates').createReport;
createReport({
                template: fs.readFileSync(filepath),
                data: body,
                noSandbox: true,
                literalXmlDelimiter: '||',
                processLineBreaks: true
}).then(function(report) {
 fs.writeFileSync(filepath , report);
 ... });
```

I'm using `IMAGE `command.

**Comment by jjhbw** [comment]
Fixed in master by 636db40f1f7a8cbb1d8f303d0f59fd69641ee837. This problem was caused by my naive fix for #217 (5f2da6711f2ee33248a131a9311197176ee5719f).

**Comment by jjhbw** [comment]
Fix released in v4.7.0. Should be on NPM shortly.
